security/pam-u2f/files/extend-nouserok.patch

```diff
@@ -0,0 +1,22 @@
+--- pam-u2f.c
++++ pam-u2f.c
+@@ -224,9 +224,16 @@ int pam_sm_authenticate(pam_handle_t * pamh, int flags, int argc,
+       get_devices_from_authfile(cfg->auth_file, user, cfg->max_devs,
+                                 cfg->debug, devices, &n_devices);
+   if (retval != 1) {
+-    DBG(("Unable to get devices from file %s", cfg->auth_file));
+-    retval = PAM_AUTHINFO_UNAVAIL;
+-    goto done;
++    if (cfg->nouserok) {
++      DBG(("Unable to get devices from file %s but nouserok specified. Skipping authentication", cfg->auth_file));
++      retval = PAM_SUCCESS;
```

Comment:
I see. But why should we add this patch instead of waiting for upstream to release the next version (which would include this patch by default)? Sorry if I'm being pedantic 😄